nitter/data/nitter/entrypoint.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
```

Comment:
It looks like this entrypoint script is never called/used as the `docker-compose.yml` has `/entrypoint.sh` defined as the `entrypoint` which looks like the default implementation included in the image already (https://github.com/goodtiding5/docker-nitter/blob/01a2e49c7a050edbe4b06bd49bb97b12fcac2a17/Dockerfile#L42)